src/main/java/mate/academy/bookstore/model/User.java

```diff
@@ -0,0 +1,41 @@
+package mate.academy.bookstore.model;
+
+import jakarta.persistence.Column;
+import jakarta.persistence.Entity;
+import jakarta.persistence.GeneratedValue;
+import jakarta.persistence.GenerationType;
+import jakarta.persistence.Id;
+import jakarta.persistence.Table;
+import jakarta.validation.constraints.NotBlank;
+import lombok.Getter;
+import lombok.Setter;
+import org.hibernate.validator.constraints.Length;
+
+@Entity
+@Getter
+@Setter
+@Table(name = "users")
+public class User {
+    @Id
+    @GeneratedValue(strategy = GenerationType.IDENTITY)
+    private Long id;
+
+    @NotBlank
```

Comment:
Let's use one of annotations NotBlank or Column(nullable = false) 
https://www.baeldung.com/hibernate-notnull-vs-nullable

Reply:
Got your choice) 
But, we still need email = unique